### PR TITLE
[#24] form to choose ingredient

### PR DESCRIPTION
**Pull request by CallMeSandra**
This form is getting ingredients from database and allows user to choose one of them. 
@jeathli please check

connected to #24

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class HomeController < ApplicationController
```
Comment (wozane) [inline app/controllers/home_controller.rb]: I'm not sure about the controller name. IMO it should be IngredientsController.

Comment (CallMeSandra) [inline app/controllers/home_controller.rb]: I'm sure about this one :)

Comment (gotar) [inline app/controllers/home_controller.rb]: Yeap you can drop Home controller and replace by Ingidients controller with index show

**Comment by jeathli** [comment]
👍

**Comment by CallMeSandra** [comment]
@jeathli I've changed name of controller, check please :)

**Comment by CallMeSandra** [comment]
@gotar

**Comment by gotar** [comment]
+1

**Pull request merged**